yourcfp/users/models.py

```diff
@@ -1,3 +1,13 @@
+
+from django.contrib.auth.models import AbstractUser
 from django.db import models
 
-# Create your models here.
+class User(AbstractUser):
+    is_speaker = models.BooleanField(default=False)
+    is_organizer = models.BooleanField(default=False)
+
+class Speaker(models.Model):
+    user = models.OneToOneField(User, on_delete=models.CASCADE)
+
+class Organizer(models.Model):
+    user = models.OneToOneField(User, on_delete=models.CASCADE)
```

Comment:
Roles

- Admin
- Organisers
- Reviewers
- Speakers
- Sponsors
- Volunteers

Reply:
@tapaswenipathak 
Do we really need to create separate roles, we can create tables and assign permissions.
Reviewers, Volunteers can be relates to organizer table.
Sponsors can be related to event model.